### PR TITLE
7477 Update CONTRIBUTING.md to not mention missing scripts

### LINKED ISSUE #7477: CONTRIBUTING.md meantions some scripts that dont exits anymore
CONTRIBUTING.md meantions some scripts that dont exits anymore like `composer complete-check` and `composer fix-cs` but these scripts aren't in `composer.json` anymore since: https://github.com/rectorphp/rector/commit/d56e7982d04a8ed2d1c66609061b2b6152048bb7

### PR DESCRIPTION
**Pull request by JohJohan**
Fixes https://github.com/rectorphp/rector/issues/7477

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,23 +30,10 @@ Do you want to **contribute a failing test**? [This tutorial will show you how](
...
 ## Preparing Pull Request
 
-3 steps will make your pull-request easy to merge:
-
 - **1 feature per pull-request**
 - **new features need tests**
-- CI must pass... you can mimic it locally by running
-
-    ```bash
-    composer complete-check
```
Comment (samsonasik) [inline CONTRIBUTING.md:40]: The script is exists, contributing is to rector-src which the composer.json consist the script, see 

https://github.com/rectorphp/rector-src/blob/73aa7e3d68d74fc0b9b26fc543bf1207cfc8d0dd/composer.json#L116

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,23 +30,10 @@ Do you want to **contribute a failing test**? [This tutorial will show you how](
...
-    ```
-
-- Do you need to fix coding standards?
-
-    ```bash
-    composer fix-cs
```
Comment (samsonasik) [inline CONTRIBUTING.md:46]: The script is exists, contributing is to rector-src which the composer.json consist the script, see 

https://github.com/rectorphp/rector-src/blob/73aa7e3d68d74fc0b9b26fc543bf1207cfc8d0dd/composer.json#L126

**Comment by samsonasik** [comment]
Please see point no 1, the repository to contribute is `rector-src`, that's why you can't find in rector/rector, see https://github.com/rectorphp/rector-src/blob/main/CONTRIBUTING.md#preparing-local-environment

File: CONTRIBUTING.md (hunk 1 above)
Comment (JohJohan) [reply CONTRIBUTING.md:40]: Your right! is there a reason why it isn't in https://github.com/rectorphp/rector/blob/main/composer.json ?

I was working there on a fix then found out i should fix it elsewhere. But these commands are not in rectorphp/rector

Comment (samsonasik) [reply CONTRIBUTING.md:40]: rectorphp/rector is generated content, you need to contribute it here instead.

Comment (JohJohan) [reply CONTRIBUTING.md:40]: Okay then we can close this pull request and this issue: https://github.com/rectorphp/rector/issues/7477

**Pull request closed without merging**